modules/nf-core/bowtie/build/meta.yml

```
@@ -15,13 +15,23 @@ tools:
       arxiv: arXiv:1303.3997
       licence: ["Artistic-2.0"]
 input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing information about the genome fasta
+        e.g. [ id:'test' ]
   - fasta:
       type: file
       description: Input genome fasta file
 output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing nformation about the genome fasta
+        e.g. [ id:'test' ]
   - index:
       type: file
-      description: Bowtie genome index files
+      description: Folder containing bowtie genome index files
```

```suggestion
      description: bowtie genome index files
```
If I see it correctly, you get all the `ebwt` files in `index`, not just the `bowtie` folder containing them

---

They are all inside the `bowtie` folder, so the downstream process receives the whole folder. Not sure why the nf-test doesn't show that on the md5sums